### ISSUE TITLE
Breakpoint moves with copy line up/down action

### ISSUE THREAD

**Issue by eyalroth** [microsoft/vscode#75426]
**Version:**
- VSCode Version: 1.35.0
- OS Version: Windows 10.0.17134.799

**Steps to Reproduce:**
1. Create a new code file (of any supported language).
2. Write a few lines of code.
3. Place a break-point on a line.
4. Copy the line using _Copy Line Up_ or _Copy Line Down_ actions.

**Bug:**
The break-point moves with the copied line. It should stay.

**Does this issue occur when all extensions are disabled?** Yes

**Comment by alexandrudima** [comment]
This is as designed... The copy line down action is implemented by inserting the line text above it, so that's why the breakpoint moves. We will not reimplement the action just to accommodate this case, and it is IMHO not clearly wrong...

**Comment by eyalroth** [comment]
@alexandrudima This really doesn't seem to be as designed but rather a clear oversight, and for multiple reasons:
1. It's a copy action -- _not_ a move action -- and the normal copy+paste action doesn't move any breakpoints around. Even cutting a line doesn't move the breakpoint.
2. The are more bugs produces by this behavior; for instance, after completing the reproduction steps  described in this issue, if one deletes said line the breakpoint will move to the next (non duplicated) line instead of the original one, and if one then tries to undo these steps, the breakpoint will remain on the next line which has nothing to do with the original one.
3. Both IntelliJ and Eclipse -- two of the most popular IDEs out there -- do not exhibit this behavior.
4. Copying a line via the _Copy Line Up/Down_ action is intended to create a new line which is similar to the original one. It's a means of creating new content which looks like old content. It is not meant to alter the original content, which goes for its state as well.
5. This behavior actually goes further than just breakpoints. For instance, the bookmark of a line marked with [the bookmarks extension](https://github.com/alefragnani/vscode-bookmarks) will behave similar to how a breakpoint does, which makes no sense. Duplicating a line definitely shouldn't move the bookmark around, as it belongs to the original line which is not about to be modifies (unlike the newly duplicated line).

**Comment by alefragnani** [comment]
I wouldn't say that keeping the breakpoint, bookmark, or any other decoration/data in the original line is correct either, because the command is `Copy Line Up/Down`, so users could expect that it should _copy_ the breakpoint/bookmark/etc to the next/previous line, duplicating it. 

I'm not sure if copying/cutting the entire line _shouldn't_ bring its breakpoint/bookmarks (sometimes I hoped it should). But to this, the API should be updated to describe the _original_
`Line`/`Range`.

IMHO, I prefer the way it works today, even with this _naming conflict_. The Bookmarks extension, at least, should remain this way.

**Comment by alefragnani** [comment]
Yes, the Bookmarks extension has its own storage, and a custom _sticky_ algorithm to update the Bookmark position (line) when the user changes the file.

Technically speaking, I guess (not in front of my computer) it works this way because the `Copy Line Down` command, in fact, doesn't add a copy of the current line, in the next line (down). Instead, it adds the copy _above the current line_, moving the current line, one line below. Doing so, the Bookmark _follows_ the original line, going one line below, just like it does when you hit `Return` at the beginning of the current line.
